src/config.ts

```diff
@@ -30,6 +35,18 @@ export const LOCALSTACK_ENDPOINT: string | undefined = loadOptionalEnvVar(
   "LOCALSTACK_ENDPOINT",
 );
 
+// Rate limiting
+
+export const lowRateLimiterConfiguration: TokenBucketConfiguration = {
+  capacity: 500,
+  numberOfSecondsBeforeRefill: 60,
+};
+
+export const highRateLimiterConfiguration: TokenBucketConfiguration = {
+  capacity: 1000,
```

Comment:
The high capacity token bucket is not going to be used for now as the web app feature to enable it is not a priority at the moment.